lib/TPP/RewriteToBatchReduceGemm.cpp

```diff
@@ -108,7 +108,7 @@ static LogicalResult checkVNNIAccessPatterns(linalg::LinalgOp linalgOp) {
   SmallVector<AffineMap> expectedMaps;
   bindDims(linalgOp.getContext(), r1, r2, p4, p5, r3);
   auto blockingFactor =
-      vnni::utils::getVNNIBlockingFactor(linalgOp->getOperands()[0].getType());
+      vnni::utils::getVnniBlockingFactor(linalgOp->getOperands()[0].getType());
```

Comment:
Both `checkVNNIAccessPatterns` and `checkAccessPatterns` are very similar and could be de-duplicated.